cmd/server-main.go

```diff
@@ -380,9 +380,8 @@ func serverMain(ctx *cli.Context) {
 	globalNotificationSys = NewNotificationSys(globalServerConfig, globalEndpoints)
 
 	// Initialize notification system.
-	if err = globalNotificationSys.Init(newObject); err != nil {
-		logger.Fatal(err, "Unable to initialize notification system")
-	}
+	// Ignoring the errors
+	_ = globalNotificationSys.Init(newObject)
```

Comment:
The error should be logged here.

Reply:
We already log the notification errors before 

![image](https://user-images.githubusercontent.com/5410427/49206046-ca74c000-f3d6-11e8-85b2-b00e546df704.png)

Should we still add `Unable to initialize notification config` ?

Reply:
:+1: